examples/ultralytics-yolov8/README.md

```diff
@@ -0,0 +1,21 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Ultralytics YOLOv8 DeepSparse Inference
+
+Ultralytics YOLOv8 integration and example has been moved to the main
+deepsparse package.  See docs, code, and examples at
+https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/yolov8.
```

Comment:
Let's get this to be a clickable link to easily navigate the user

Reply:
it is clickable